Sources/BraintreeThreeDSecure/BTThreeDSecureRequest.m

```diff
@@ -53,6 +53,7 @@ - (instancetype)init {
     self = [super init];
     if (self) {
         _versionRequested = BTThreeDSecureVersion2;
+        _cardAddChallengeRequested = BTThreeDSecureCardAddChallengeUnspecified;
```

Comment:
```suggestion
```

We can omit this since ObjC will default the enum to the first value option in the ENUM list.

Reply:
Cool good to know 👍